_data/callouts/home.yml

```diff
@@ -1,14 +1,27 @@
+---
 style: is-light
 items:
-  - title: Save the Date!
-    subtitle: The second annual US-RSE Conference will take place October 15-17, 2024 in Albuquerque, NM
-    link: http://us-rse.org/usrse24/
-    link_name: Full Details
-  - title: Get Involved
-    subtitle: Community and working group events can be found on the Events page
+  - title: Read our latest newsletter
+    subtitle: >
+      In this monthly newsletter, we share recent, current, and planned
+      activities of the US-RSE Association, and related news that we think is
+      of interest to US-RSE members.
+    link: https://us-rse.org/newsletters/
+    link_name: Newsletters
+  - title: RSE Career Guide
+    subtitle: A joint publication from US-RSE and the IEEE Computer Society
     link: https://us-rse.org/events/
```

Comment:
I believe this link needs to be updated, as it is currently pointing to the general events page. I'm not sure if we have a post about the RSE career guide paper, but I think this is the actual guide: https://zenodo.org/records/10073233.

Reply:
Thanks, good catch. I updated the link.